src/client/cmd/common_cli.cpp

```diff
@@ -102,3 +102,23 @@ mp::ReturnCode cmd::standard_failure_handler_for(const std::string& command, std
 
     return return_code_for(status.error_code());
 }
+
+bool cmd::update_available(const multipass::UpdateInfo& update_info)
+{
+    return update_info.version() != "";
```

Comment:
Do we need this? Is there no other way to check if `reply.update_info()` is valid? IIUC it should only be the case if the daemon decided it's time?

Reply:
I (ab)used the fact that an empty update version string implies no update message to be shown.

The server didn't set the property at all, so gRPC sets a default value of empty string.
https://developers.google.com/protocol-buffers/docs/proto3#default

I'm not aware of any other way to check if valid, aside from doing what Chris did with metrics and add an explicit boolean "metrics_pending"

Reply:
Ack.